README.md

```diff
@@ -210,11 +210,32 @@ app.use(limitMiddleware.middleware(options, function(req, res, next) {
 }));
 ```
 
+Include ratelimit response headers:
+
+```javascript
+var options = {
+  headers: true
```

Comment:
I would suggest we go with the simpler solution for the time being. If there's a lot of feedback about more granularity, then I think it makes sense to break this out further. Thanks for the suggesting.